### DISCUSSION TITLE
Tell me plzz about how to use Performances?

### DISCUSSION THREAD

**Discussion by DutchBigEd** [probonopd/MiniDexed#429] [Q&A] [answered]
Ok, got my PI4b up and running the 8 instances of a DX7, I know how to navigate the Banks and Voices, but what about Performances?
I have googled I guess for hours, but no clear and/or understandable answer.
So here's my question: Where can I read up about how to work with performances in the minidexed DX7?

Thank you,
Ed

**Comment by Banana71** [comment, marked as answer]
Hi Ed,

Load performance:
- Start from the main menu, double click on the Rotery encoder. You should now see (TG1) on the bottom left of the display.
- Turn the encoder 9 clicks to the right until you see (Performance) in the display.
- Click on the encoder to enter the Load Performance menu (Load)
- Then press the encoder again (default).
- Now you can preselect a performance by turning the encoder and the performance is loaded by pressing the encoder.

I hope that answers your question. All menu items are listed in the wiki: [Wiki-Menu](https://github.com/probonopd/MiniDexed/wiki/Menu)
You can find more performances here: https://github.com/probonopd/MiniDexed/discussions/414



Peter

**Reply by DutchBigEd** [reply]
Thank you Peter, I did read the wiki about this but this is not exactly what I meant..
Let me elaborate on my question:
What exactly does a performance file do? Does it change the way a loaded voice sounds, or does it creates sounds by itself instead of a voice?
The voices should be named started with 00000n_voice.syx, is this also true for performance names and do they have to have the same name as the voices?

Thanks,
Ed

**Reply by Onefelloff** [reply]
Hi, as I understand it a Performance is just a snapshot of the way minidexed is set up at the time, it relies on a combination of voices from the banks and all of the other setting, such as pan, defund, MIDI channels etc. 

I’ve only saved performances from within Minidexed. So I’m not sure about the file names.

**Reply by Banana71** [reply]
The minidexed is 8-parts multitimbral and each of these tone generators (TG1 - TG8) can be parameterized. All settings that can be made on the minidexed are saved in the Performance.ini. (and someone more) Voice, Bank, PAN, Detune, Volume, Cutoff, MidiChannel,.....
Even the changes to the voice data are saved here. 
The naming of the performance files is independent of the voice names. It would certainly be possible to have 8 different voice names in one performance. [Wiki - Files - performance.ini ](https://github.com/probonopd/MiniDexed/wiki/Files#performanceini)
It's actually like the freeze function in computer games, with which you can always jump back to a saved game.

e.g. Barock.ini
[https://github.com/probonopd/MiniDexed/blob/main/performance/000040_Barock.ini](https://github.com/probonopd/MiniDexed/blob/main/performance/000040_Barock.ini)

**Comment by DutchBigEd** [comment]
Thank you!
